scripts/globals/status.lua

```diff
@@ -2443,6 +2443,11 @@ xi.mobMod =
     ALLI_HATE           = 68, -- Range around target to add alliance member to enmity list.
     NO_LINK             = 69, -- If set, mob cannot link until unset.
     NO_REST             = 70, -- Mob cannot regain hp (e.g. re-burrowing antlions during ENM).
+    DRAW_IN_INCLUDE_PARTY     = 71, -- this will cause the mob's draw-in to also affect all party and alliance members
```

Comment:
We want to take a different direction than this. we're planning to move the bulk of draw in to lua via a combination of listener, mixin, and binding. While I appreciate the binding using multiple mob mods to control its behavior is cumbersome. 

We're going to handle it the behaviors cotnrols and current plan is to use parameters on a mixin to set behavior. Kinda like how 2hrs get controlled by params.